pkg/deploy/identity-provider/identity_provider.go

```diff
@@ -31,8 +31,14 @@ const (
 
 // SyncIdentityProviderToCluster instantiates the identity provider (Keycloak) in the cluster. Returns true if
 // the provisioning is complete, false if requeue of the reconcile request is needed.
-func SyncIdentityProviderToCluster(deployContext *deploy.DeployContext, cheHost string, protocol string, cheFlavor string) (bool, error) {
+func SyncIdentityProviderToCluster(deployContext *deploy.DeployContext) (bool, error) {
 	instance := deployContext.CheCluster
+	cheHost := instance.Spec.Server.CheHost
+	protocol := "http"
+	if instance.Spec.Server.TlsSupport {
```

Comment:
It isn't depreciated yet.
Instead of passing `cheHost, protocol, cheFlavor` I've just redefine it inside function because I've already passed `deployContext` which contain all necessary data.

Reply:
Is it a mistake that it is commented as `deprecated` in the API then: https://github.com/eclipse/che-operator/blob/master/pkg/apis/org/v1/che_types.go#L135 ?

Reply:
Yes, is it deprecated, but the code base still exists. 
we have the pr https://github.com/eclipse/che-operator/pull/320 but it is postponed.

Reply:
ok